### PR TITLE
Hydradx collator setup how-to guide

### PR DESCRIPTION
**Pull request by fakirAyoub**

### CODE REVIEW

File: docs/run_hydradx_collator.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+```bash
+sudo adduser hydra
+sudo usermod -aG sudo hydra
+su - hydra
+```
+
+## Download and configure the `hydradx` binary
+
+Pick a release, for instance, let's use the latest release to date (11.2.1) from our assets repository:
```
Comment (lumir-mrkva) [inline docs/run_hydradx_collator.md:18]: you have to use release 12+, 11 is still snakenet

File: docs/run_hydradx_collator.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Description=hydradx validator
+[Service]
+Type=exec
+User=hydra
+ExecStart=/usr/local/bin/hydradx \
+    --name YOUR_COLLATOR_NAME \
+    --base-path /var/lib/hydradx \
+    --prometheus-external \
+    --collator \
+    --telemetry-url "wss://telemetry.hydradx.io:9000/submit/ 0" \
```
Comment (lumir-mrkva) [inline docs/run_hydradx_collator.md:49]: this has to be in the collator portion (after `--`)

File: docs/run_hydradx_collator.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+sudo systemctl enable hydradx-collator
+sudo systemctl start hydradx-collator.service
+```
+
+And voilà ! Your node should be up and running now. Make sure your `hydra` user has the necessary permissions to access your `base-path` and key file.
+
+If you need to troubleshoot your running service, you can use the `journalctl` with `-f` option for tailing command as follows:
+
+```bash
+journalctl -u hydradx-collator -f
```
Comment (lumir-mrkva) [inline docs/run_hydradx_collator.md:85]:
```suggestion
journalctl -fu hydradx-collator
```

File: docs/run_hydradx_collator.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+Fill in the fields:
+
+- *using selected account*: select your Controller account;
+- *submit the following extrinsic*: select `session` on the left side and `setKeys` on the right;
+- *keys*: enter your session key you just generated;
+- *proof*: `0`;
+
+## What's next?
+
+That's it, you're done. Let us know, and we will add your node to the collator set.
```
Comment (lumir-mrkva) [inline docs/run_hydradx_collator.md:115]: not, yet, you have to make sure that your node is fully synced, before we can add you

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

File: docs/run_hydradx_collator.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+Description=hydradx validator
+[Service]
+Type=exec
+User=hydra
+ExecStart=/usr/local/bin/hydradx \
+    --name YOUR_COLLATOR_NAME \
+    --collator \
+    -- \
+    --execution wasm \
+    --prometheus-external \
```
Comment (lumir-mrkva) [inline docs/run_hydradx_collator.md:49]: prometheus external has to be in parachain part if you want to export parachain start and not relaychain, it is worth noting that you have to set up separate port when you want to export both

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

File: docs/run_hydradx_collator.md (hunk 5 above)
Comment (fakirAyoub) [reply docs/run_hydradx_collator.md]: Done.

**Review by vgantchev — APPROVED** [review]

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**